api/utils/BaseException.js

```diff
@@ -1,7 +1,18 @@
 //TODO: namespace?
 
 class BaseException extends Error {
-  constructor() {}
+  /**
+   * @class BaseException
+   * @constructor
+   * @private
+   * @param {Number} code Error code
+   * @param {String} message Error message
+   */
+  constructor(code = 500, message = 'unknown error') {
```

Comment:
named parameter로 하는게 좋을 것 같습니다.
```suggestion
  constructor({ code = 500, message = 'unknown error' }) {
```

Reply:
오..이런게 있군요 바꿔보겠습니다.

Reply:
<img width="505" alt="스크린샷 2022-03-10 오전 11 44 58" src="https://user-images.githubusercontent.com/76928178/157578701-36893941-8497-4ea5-b293-6e6d8ddc5295.png">
말씀하신 대로 named parameter로 작성해보니 JSDoc에 parameter의 타입에 대한 명시를 1:1로 줄수가 없어
<img width="484" alt="스크린샷 2022-03-10 오전 11 47 58" src="https://user-images.githubusercontent.com/76928178/157578974-0cf81c6a-50f3-4a61-8212-726c8eb68fc7.png">
이렇게 작성하게 되어서..제 생각에는 기존 방식대로 parameter를 작성하는게 나을거 같습니다만, 이런 사소한 단점보다 장점이 더 큰 것인가요?